Makefile

```diff
@@ -1,6 +1,9 @@
 BUILD_TOOL = xcodebuild
-BUILD_SCHEME = SQLite Mac
```

Comment:
It wasn't a default, it was just ignoring whatever you passed in and always using `SQLite Mac` for the scheme.

But yes, I agree that having a default to make it easier to run quick tests from the command line would be a good change to make.